### ISSUE TITLE
[Avatar] Setting "picture" as not readable for a11y readers

### ISSUE THREAD

**Issue by jpzwarte** [sl-design-system/components#3095]
### Provide a general summary of the issue here

Is it possible to modify sl-avatar so that the \"picture\" element is treated by accessibility readers (a11y) as a graphic representation? Currently, when initials are displayed inside the \"picture\", screen readers read those initials aloud — which, according to a11y guidelines, is unnecessary and disrupts the user experience.

### :thinking_face: Expected Behavior?

\"picture\" element is treated by accessibility readers (a11y) as a graphic representation, and is ommitted from reading when it contains initials.

### :hushed: Current Behavior

Currently, when initials are displayed inside the \"picture\" element, screen readers read those initials aloud — which, according to a11y guidelines, is unnecessary and disrupts the user experience.

### 👤 Your name

Konrad Erbetowski

### 🧢 Your Product/Team

CFP Learning Analytics
